docker-compose.yaml.sample

```diff
@@ -0,0 +1,26 @@
+version: '3.7'
+
+services:
+  clair:
+    image: quay.io/coreos/clair:latest
```

Comment:
The issue i see here is by pulling :latest the user will not always run the Clair version they may think.

Reply:
I totally agree. My reason for using `:latest` is to be consistent with the `Docker` part of the Documentation where `:latest` is already used. Do you think it would be better to change to `:v2.1.2` or even ':v3.0.0-pre1`?

Reply:
In general using `latest` is not the best thing, unless it's for development purposes. This is a sample file as well, comments can be added to explain prod vs dev environments?
Removing the latest and setting a version would put some more works on the documentations maintainers, unless some placeholder is put? `<version needed>`